geode-core/src/main/java/org/apache/geode/distributed/AbstractLauncher.java

```diff
@@ -502,7 +507,7 @@ protected static Integer identifyPid() {
       }
     }
 
-    // TODO refactor the logic in this method into a DateTimeFormatUtils class
+    // consider refactoring the logic in this method into a DateTimeFormatUtils class
```

Comment:
I'd remove this and the line 510 comments altogether.  If the suggestion has value, we should open a ticket for it.

Likewise `LocatorLauncher:382`, `ServerLauncher:523`